src/hooks/useKeyEvent/index.ts

```diff
@@ -1,18 +1,44 @@
-import { useCallback, useRef } from "react";
+import { RefObject, useCallback, useRef } from "react";
 import useEventListener from "../useEventListener";
+import { GenericEventCallback } from "../../types/events";
 
 const CTRL_OR_META = "ctrlOrMetaKey";
 
-const checkModifierInEvent = (event, modifier) => {
-  if (event[modifier]) return true;
-  if (modifier === CTRL_OR_META) {
+enum Modifiers {
+  ALT = "altKey",
+  META = "metaKey",
+  CTRL = "ctrlKey",
+  SHIFT = "shiftKey",
+  CTRL_OR_META = "ctrlOrMetaKey"
+}
+
+const checkModifierInEvent = (event: KeyboardEvent, modifier: Modifiers) => {
+  if (modifier === Modifiers.CTRL_OR_META) {
     return event.ctrlKey || event.metaKey;
   }
-  return false;
+
+  return event[modifier];
 };
-const checkWithoutModifierInEvent = event => {
-  return !Object.values(useKeyEvent.modifiers).some(m => event[m]);
+const checkWithoutModifierInEvent = (event: KeyboardEvent) => {
+  return !Object.values(useKeyEvent.modifiers).some((m: Modifiers) => {
+    if (m === CTRL_OR_META) return true;
+    return !!event[m];
+  });
 };
+
+export interface UseKeyEvent {
```

Comment:
It's hard to understand from the interface's name what's it used for - is it the hook's return value? arguments shape?

(I can see 2 lines below it's the arguments shape, but still).